scripts/prettier.sh

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
```

Comment:
why removing this? 

Check if scripts is needed at all after that.

Check if there are references to prettier.sh somewhere in docs.

Reply:
I'm pretty sure it was an old way to do what we have in yarn scripts now. Do you want it back just in case? It's not used anywhere in the repo anymore.

I had forgotten to update the contributing docs page though, just did in 38ec8ae.

Reply:
I'm not sure yarn scripts can do that (especially the way they are written now - only run on staged files).I think the idea was to have a mechanism to run prettier on a specific files/files w/o running `git commit` or even `yarn something` (which is not possible now since it's only prettier-staged which is the same as running git hook?)

Reply:
Is it a common need to run prettier on arbitrary files in our repo? I'm not sure it is but I would just document Prettier usage examples in the docs contrib guide such as `yarn prettier --write '{pages,src}/**/*.{js,jsx,md}'`. But is it goal of this repo to teach Prettier usage?

Reply:
I added the note in 9ccfb51 (and made some other improvements about this in general). What do you think?

Reply:
> Is it a common need to run prettier on arbitrary files in our repo?

At some point in time the hooks that fix the formatting were not working properly. The reason for writing this script was to merge contributions from community even if they were not 100% conforming to the formatting rules, then run this script manually once in a while to correct any formatting issues. I am not sure if this is needed anymore.

But I think that it still might be useful to be able to check and fix the files before staging them.

> I would just document Prettier usage examples in the docs contrib guide such as `yarn prettier --write '{pages,src}/**/*.{js,jsx,md}'`

If you write a small script that does this, then there would be no need to document its usage.

Reply:
> If you write a small script that does this, then there would be no need to document its usage.

Yes, but that's not the only possible usage of Prettier, it could be something else you want to do (a specific folder, only md files, only md and jss, etc.) and having an entire sh script to wrap a simple `prettier` call with a few arguments seems overkill to me. Check out the note about using Prettier via `yarn` I added to the docs contrib guide. I hope this, the link to `pettier` usage, and the `js,jsx,md` example should be enough 🙂

Reply:
k, let's simplify this and see if we get some feedback that something is not clear/convenient.